src/datastructures/nalgebra.rs

```diff
@@ -8,6 +9,27 @@ use super::{AdjacencyMatrix, Edge};
 #[derive(Debug, PartialEq)]
 pub struct NAMatrix(pub DMatrix<f64>);
 
+impl NAMatrix {
+    /// Create from set of x/y points
```

Comment:
```suggestion
    /// Create from set of x/y points, using the euclidean distance
```

make it more explicit what metric you are using